app/models/user.rb

```diff
@@ -116,7 +116,7 @@ def self.new_with_session(params, session)
 
   def use_default_photo_pic?
     self.profile_photo_url ==
-        'https://fbcdn-profile-a.akamaihd.net/hprofile-ak-ash4/c178.0.604.604/s160x160/252231_1002029915278_1941483569_n.jpg'
+        'http://coursemology.s3.amazonaws.com/public/default_profile_pic.png'
```

Comment:
hey @wangqiang1208, this isn't very DRY. It's repeated twice. Do you think you can extract this somewhere?